docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -72,156 +305,132 @@ emulators:
 
 When an ID is specified, it supersedes the other settings.
 
-__Type__: *[]object*
-
-__Example__:
 ```yaml
 devices:
   - name: "Google Pixel.*"
     platformVersion: 8.1
   - id: Google_Pixel_2_real_us
 ```
+---
 
-### `id`
-
-__Description__: Request a specific device by its id. You can look up device IDs in the Sauce Labs app or using our [Get Devices API request](https://docs.saucelabs.com/dev/api/rdc#get-devices).
-
-__Type__:  *string*
+#### `id`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Example__:
+Request a specific device for this test suite by its ID. You can look up device IDs in the Sauce Labs app or using our [Get Devices API request](https://docs.saucelabs.com/dev/api/rdc#get-devices).
 
 ```yaml
         id: Google_Pixel_2_real_us
 ```
+---
 
-### `name`
-
-__Description__:  Request a device by its name.
-
-__Type__:  *string*
+#### `name`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Example__:
+Find a device for this test suite that matches the device name or portion of the name, which may provide a larger pool of available devices of the type you want.
 
-Using complete name:
-```yaml
+```yaml title="Use Complete Name"
       - name: Google Pixel 4 XL
 ```
 
-Using pattern matching:
-```yaml
+```yaml title="Use Pattern Matching"
         name: Google Pixel.*
 ```
+---
 
-### `platformVersion`
-
-__Description__: Request that the device matches a specific platform version. You can use the `*` wildcard to denote flexibility.
-
-__Type__:  *string*
+#### `platformVersion`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Example__:
+Request that the device matches a specific platform version. You can use the `*` wildcard to denote flexibility.
 
 ```yaml
         platformVersion: 8.*
 ```
+---
 
-### `options`
-
-__Description__:  Further specify certain device attributes within the pool of devices that match the `name` and `version` criteria.
-
-__Type__: *object*
+#### `options`
+<p><small>| OPTIONAL | OBJECT |</small></p>
 
-#### `carrierConnectivity`
+A parent property to further specify desired device attributes within the pool of devices that match the `name` and `version` criteria.
 
-__Description__: Request that the matching device is also connected to a cellular network.
+---
 
-__Type__:  *bool*
+##### `carrierConnectivity`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
 
-__Example__:
+Request that the matching device is also connected to a cellular network.
 
 ```yaml
        carrierConnectivity: true
-
 ```
+---
 
-#### `deviceType`
-
-__Description__:  Request that the matching device is a specific type of device. Values:  `ANY`, `TABLET`, or `PHONE`.
-
-__Type__:  *string*
+##### `deviceType`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Example__:
+Request that the matching device is a specific type of device. Valid values are:  `ANY`, `TABLET`, or `PHONE` (case-sensitive).
```

Comment:
There was a discrepancy between Espresso and XCUITest (Espresso was case-sensitive whereas XCUITest was not). See https://github.com/saucelabs/saucectl/pull/353

When this will be merged/release, case-sensitiveness will not be true anymore.

```suggestion
Request that the matching device is a specific type of device. Valid values are:  `ANY`, `TABLET`, or `PHONE`.
```